.github/workflows/osx.yml

```diff
@@ -0,0 +1,47 @@
+---
+name: macOS build and test AWS Encryption SDK for C
+
+on:
+  pull_request:
+    branches:
+      - master
+
+
+jobs:
+  OSX:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        os: [macOS-10.14, macos-latest]
+        ssl_ver: [openssl@1.1]
```

Comment:
Bytes are cheap. Can we just spell this out? ie: `openssl_version`

Reply:
added.